### DISCUSSION TITLE
MIMIC-IV Demo Dataset substitute

### DISCUSSION THREAD

**Discussion by BPDanek** [sunlabuiuc/PyHealth#86] [Ideas]
Since access to MIMIC-IV sample data requires authorization from PhysioNet, for the purpose of the demo and example texts does it make sense to use this variant of the dataset instead? They are from the same author? https://physionet.org/content/mimic-iv-demo/2.2/

Pyhealth users may not be interested in the package if there are external barriers to using it.

**Comment by BPDanek** [comment]
To use the dataset, download it, and unzip it with archive utility (mac). Within the `hosp` directory you will find the `.csv` files which are supported for parsing into datasets. I had to install `gzip` (https://formulae.brew.sh/formula/gzip) to unpack these files since archive utility doesn't handle the `.gz` files by default. You can just unpack the files you'll use in your tables (in the example below it's the default tables and diagnosis_icd, procedures_icd, labevents)

```
dataset = MIMIC4Dataset(
        root=ROOT,
        tables=["diagnoses_icd", "procedures_icd", "labevents"],
        code_mapping=CODE_MAPPING,
        dev=DEV,
        refresh_cache=False
    )
```

**Reply by BPDanek** [reply]
Since this is a demo dataset, you may be able to put it behind one of the googleapi links and allow really easy usage

**Reply by ycq091044** [reply]
Done! We already linked to the demo dataset https://physionet.org/content/mimic-iv-demo/2.2/ and uploaded it to googleapi.

**Comment by netanelcyber** [comment]
I'm working on repo using MIMIC IV #1154
